ci/verify.sh

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
-./vendor/bin/php-cs-fixer fix ./src
```

Comment:
There was no point of running the CS Fixer before PHPCS.

Reply:
shouldn't you move it down then? btw imho this should be executed in "dry run" mode

Reply:
@IonBazan ping

Reply:
@Kryniol it already is. Check the file again :smile: